website/static/js/comment.js

```diff
@@ -291,6 +511,50 @@ var CommentModel = function(data, $parent, $root) {
         self.author = self.$root.author;
     }
 
+    self.editableContent = ko.computed(function() {
+        var content = self.content();
+        if (!content) {
+            content = '';
+        }
+        var regex = /\[(@|\+)(.*?)\]\(\/([a-z\d]{5})\/\)/;
+        var matches = content.match(/\[(@|\+)(.*?)\]\(\/([a-z\d]{5})\/\)/g);
+        if (matches) {
+            for (var i = 0; i < matches.length; i++) {
+                var match = regex.exec(matches[i]);
+                var atwho = match[1];
+                var guid = match[3];
+                var mention = match[2];
+
+                content = content.replace(match[0], '<span class="atwho-inserted" contenteditable="false" data-atwho-guid="'+ guid + '" data-atwho-at-query="' + atwho + '">' + atwho + mention + '</span>');
+            }
+        }
+        return content.replace(/\x0D\x0A/g, '<br>');
+    });
+
+    self.editedContent = ko.computed(function() {
+        var content = self.content();
+        if (!content) {
+            content = '';
+        }
```

Comment:
I don't see where we're ever setting `content()` to `null` or `undefined` (though I do find a [case](https://github.com/CenterForOpenScience/osf.io/blob/eac466dba48543c997dbcbecc4ed2955e41ae23e/website/static/js/comment.js#L449) with `replyContent`). If there's a reason to defend against that, then we can fold this down to:

```javascript
var content = self.content() || '';
```